src/main/java/seedu/address/model/event/EventTitleContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.model.event;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+
+
+/**
+ * Tests that an {@code Event}'s {@code EventTitle} matches any of the keywords given.
+ */
+public class EventTitleContainsKeywordsPredicate implements Predicate<Event> {
+    private final List<String> keywords;
+
+    public EventTitleContainsKeywordsPredicate(List<String> keywords) {
+        this.keywords = keywords;
+    }
+
+    //TODO: Once EventTitle class is added, we need to change this to
+    //TODO: StringUtil.containsWordIgnoreCase(event.getEventTitle().fullEventTitle, kewyord));
```

Comment:
Using toString() might be a better option so we don't directly access the attributes of EventTitle class though I'm aware that it's a public attribute at the moment
```suggestion
    //TODO: StringUtil.containsWordIgnoreCase(event.getEventTitle().toString(), kewyord));
```